### PR TITLE
Polish Non-Optimizing

### PR DESCRIPTION
**Pull request by jbae11**
The non-optimizing report has been polished to talk explicitly on the non-optimizing algorithm.

### CODE REVIEW

File: docs/no_numerical_exp_report.tex [hunk 1]
```diff
@@ -56,12 +61,11 @@ \section{Introduction}
 agent, deploys \texttt{Facilities} to meet the front-end and back-end demands of the 
 fuel cycle.
 
-This report describes numerical tests for non-optimizing, deterministic-optimizing and stochastic-optimizing prediction algorithms.
+This report describes numerical tests for the non-optimizing algorithm, namely \gls{ARMA} and \gls{ARCH}.
```
Comment (gwenchee) [inline docs/no_numerical_exp_report.tex]: should it be numerical experiments (we probably want to be more consistent with what we call the tests/numerical experiments). "This report describes numerical experiments to test the non-optimizing algorithm ... "

File: docs/no_numerical_exp_report.tex [hunk 2]
```diff
@@ -56,12 +61,11 @@ \section{Introduction}
...
 
 These prediction models are being developed by the University of South Carolina. 
-In this report, we discuss numerical experiments for testing the non-optimizing, 
-deterministic optimizing and stochastic optimizing methods. The numerical 
-experiments will be designed for both the once through nuclear fuel 
+In this report, we discuss numerical experiments for testing the non-optimizing methods.
```
Comment (gwenchee) [inline docs/no_numerical_exp_report.tex]: Repetition of line 64?

File: docs/no_numerical_exp_report.tex [hunk 3]
```diff
@@ -72,17 +76,86 @@ \section{Method}
 
 
 \section{Configuration}
-The user defines prototypes to be deployed for fuel facilities,
-and reactor deployment scheme.  The reactor deployment causes the demand of fuel
+The user defines prototypes to be deployed for fuel facilities, and the 
+demand equation.  The reactor deployment causes the demand of fuel
```
Comment (gwenchee) [inline docs/no_numerical_exp_report.tex]: This isn't very clear. Are you trying to say: 
The user defines what prototypes are to be deployed and the positive or negative demand growth equation for one of the commodities. For example, if the user chooses a positive demand growth equation for power, this triggers reactor deployment. The reactor deployment then causes an increase in the demand of fuel, which triggers fuel facility deployment.

File: docs/no_numerical_exp_report.tex [hunk 4]
```diff
@@ -72,17 +76,86 @@ \section{Method}
...
 \section{Algorithm Flow}
 
-The algorithm, upon entering, creates a supply chain with the fuel facilities and
-the reactor. Then, at every timestep it calculates the expected demand from each fuel cycle
-facility and makes decisions to deploy or decommission. As a reference, the time step
-execution for \Cyclus is illustrated in figure \ref{diag:time}.
+The algorithm is an \texttt{Institution}, which is a \Cyclus agent type
+that deploys and decommissions facilities. 
+
+The algorithm predicts the demand by looking at the previous demand and supply values.
```
Comment (gwenchee) [inline docs/no_numerical_exp_report.tex]: Perhaps: The algorithm predicts the demand of a commodity by looking at its previous demand and supply values.

File: docs/no_numerical_exp_report.tex [hunk 5]
```diff
@@ -72,17 +76,86 @@ \section{Method}
...
+This is done through the \texttt{TimeSeries} toolkit in \Cyclus. To implement
+this functionality, every \Cycamore archetype is configured to record their
+demand and supply whenever they occur (usually, at every \texttt{Tick} or \texttt{Tock},
+discussed later). The algorithm accesses this timeseries and makes a prediction to
+what the demand will be in the future. If the algorithm sees a lack or excess of supply,
+it deploys new facilities or decommissions existing facilities.
```
Comment (gwenchee) [inline docs/no_numerical_exp_report.tex]: " ... or decommissions existing facilities correspondingly."

File: docs/no_numerical_exp_report.tex [hunk 6]
```diff
@@ -72,17 +76,86 @@ \section{Method}
...
+
+\subsection{Time Step Execution in \Cyclus}
+As a reference, the time step execution for \Cyclus is illustrated in figure \ref{diag:time}.
+
+At \texttt{Tock}, the algorithm calculates the demand and supply for the next timestep, and calculates
+the difference. If the difference is bigger than the capacity of one facility, it
+schedules to deploy a new facility, or a decommissioning of an existing facility.
+The decommissioning takes place in the \texttt{Decommission} phase of \Cyclus, and
+the construction takes place in the \texttt{Build} phase of \Cyclus. Thus, all the
+adjustment of facilities occur prior to the next DRE.
```
Comment (gwenchee) [inline docs/no_numerical_exp_report.tex]: might wanna spell dynamic resource exchange out, because i don't think it is mentioned before this.

**Comment by katyhuff** [comment]
@gwenchee , please see https://github.com/gwenchee/ddca_numerical_exp/pull/5

**Pull request merged**